apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanStack.tsx

```diff
@@ -152,7 +147,7 @@ export const KanbanStack = forwardRef<VirtuosoHandle, IKanbanStackProps>((props,
                   draggableId={card.id}
                   index={realIndex}
                   key={card.id}
-                  isDragDisabled={!cardDraggable}
+                  isDragDisabled={!cardDraggable || isComputed}
```

Comment:
Is the field read-only permission not considered here?

Reply:
`cardDraggable` indicates whether it has read-only permissions.